js_modules/dagit/packages/core/src/assets/AssetMaterializations.tsx

```diff
@@ -130,13 +184,45 @@ export const AssetMaterializations: React.FC<Props> = ({
           <AssetMaterializationGraphs
             xAxis={xAxis}
             asSidebarSection
-            assetMaterializations={reversed}
+            assetMaterializations={grouped}
           />
         </SidebarSection>
       </>
     );
   }
 
+  const focused =
+    grouped.find((b) =>
+      params.time
+        ? Number(b.timestamp) <= Number(params.time)
+        : params.partition
+        ? b.partition === params.partition
+        : false,
+    ) || grouped[0];
```

Comment:
Does `grouped` definitely have at least one item? I believe we aren't doing anything to guard against this being undefined in TS, which I've been meaning to see if we can clean up...

Reply:
Hm that's a good question... I think for an asset that has never been materialized grouped can be empty. Let me make sure that works properly.